source/ember-inspector/container.md

```diff
@@ -1,29 +1,19 @@
-Ember.js implements dependency injection by using a container.  Every Ember
-application has a container that creates instances for you. You can
-inspect these instances using the inspector's container tab.
-
+Every Ember application has a container that maintains object instances for you. You can
+inspect these instances using the Container tab. This is useful for objects
+that don't fall under a dedicated menu, such as services.
 
 <img src="../../images/guides/ember-inspector/container-screenshot.png" width="680"/>
 
-You will see a list of object types the container has cached. Click on
-one type to see the list of all instances created by the container.
-
-In our case, we are looking at the instantiated controllers.
-
+Click on the Container tab, and you will see a list of instances the container is holding. Click on a type to see the list of all instances of that type maintained by the container.
 
 ### Inspecting Instances
 
-Click on one row to inspect an instance using the object inspector.
+Click on a row to inspect an given instance using the Object Inspector.
 
 <img src="../../images/guides/ember-inspector/container-object-inspector.png" width="680"/>
 
-To send an instance to the console, click to open the
```

Comment:
Yes, it's in the Object Inspector guide entry, so I've been taking it out elsewhere for brevity.